### PR TITLE
Support push events for action-plan-preview

### LINKED ISSUE #3481: Support push events for actions-plan-preview
**What would you like to be added**:

* Support push events for actions-plan-preview

**Why is this needed**:

To run actions-plan-preview after automatically creating a PR as follows, supporting push event is needed. The reason why this is necessary is personal access token triggers PR events after create PRs but [GITHUB_TOKEN](https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#triggering-a-workflow-from-a-workflow) doesn't do so.

```
on:
  push:
    branches: target-branch
jobs:
  create-pr:
    runs-on: ubuntu-latest
    if: "github.event.created"
    env:
      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
    steps:
      - uses: actions/checkout@v2
      - id: create-pr
        run: |
          PR=$(gh pr create --title "Update" --body "" | awk -F "/" '{print $NF}')
          echo "::set-output name=pull-request-number::$PR"
      - uses: pipe-cd/actions-plan-preview@v1.7.2
        with:
          address: ${{ secrets.PIPECD_API_ADDRESS }}
          api-key: ${{ secrets.PIPECD_PLAN_PREVIEW_API_KEY }}
          token: ${{ secrets.GITHUB_TOKEN }}
          pull-request-number: ${{ steps.create-pr.outputs.pull-request-number }}
```

### PR DESCRIPTION
**Pull request by gkuga**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #3481

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: tool/actions-plan-preview/main.go [hunk 1]
```diff
@@ -33,6 +34,13 @@ const (
 	defaultTimeout       = 5 * time.Minute
 	pullRequestEventName = "pull_request"
 	commentEventName     = "issue_comment"
+	pushEventName        = "push"
+
+	addressArgName = "address"
+	apiKeyArgName  = "api-key"
+	tokenArgName   = "token"
+	timeoutArgName = "timeout"
+	prNumArgName   = "pull-request-number"
```
Comment (nghialv) [inline tool/actions-plan-preview/main.go:43]:
```suggestion
	argAddress = "address"
	argAPIKey  = "api-key"
	argToken   = "token"
	argTimeout = "timeout"
	argPRNum   = "pull-request-number"
```

Comment (gkuga) [reply tool/actions-plan-preview/main.go]: I fixed name. Review again please.

**Comment by nghialv** [comment]
Thank you. 👍 
/lgtm

**Review by knanao — COMMENTED** [review]
Looks neats. Here you go!
/approve

**Pull request merged**